definitions/infra-awskinesisdeliverystream/golden_metrics.yml

```diff
@@ -18,7 +18,7 @@ getThroughput:
   unit: BYTES_PER_SECOND
   queries:
     aws:
-      select: sum(aws.firehose.DeliveryToElasticsearch.Bytes) + sum(aws.firehose.DeliveryToS3.Bytes) + sum(aws.firehose.DeliveryToRedshift.Bytes)
+      select: rate(sum(aws.firehose.DeliveryToElasticsearch.Bytes) + sum(aws.firehose.DeliveryToS3.Bytes) + sum(aws.firehose.DeliveryToRedshift.Bytes), 1 second)
```

Comment:
What is the reason for this change? The queries from all the instrumentation providers need to use the same value function and as well have to use the same rate. We can't just add the rate function to one of them

Reply:
@josemore is informed of that, the query is different between GM and SM and generate discrepancies

Reply:
My point is that if you use a rate for one instrumentation provider, you'll have to use a rate for the other instrumentation providers as well. Otherwise, we might get different results based on whether the entity uses one or the other instrumentation provider.

Reply:
The idea is to have GoldenMetric with rate, same as their summary-metrics https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-awskinesisdeliverystream/summary_metrics.yml#L18